engine/runtime/src/main/java/org/enso/interpreter/runtime/builtin/Builtins.java

```diff
@@ -80,25 +87,38 @@ public final class Builtins {
   private final AdditionalWarnings additionalWarnings;
   private final Builtin instrumentor;
 
+  /** Factory method to create the builtins. */
+  private static Builtins create(EnsoContext context) {
+    var fqn = QualifiedName.fromString(MethodNames.Builtins.MODULE_NAME);
+    var res = new Builtins[1];
+    var module =
+        Module.emptyWith(
+            fqn,
+            null,
+            (scopeBuilder) -> {
+              res[0] = new Builtins(context, scopeBuilder);
```

Comment:
- 1095d2a shows the idea of controlling who has access to the scope builder
- `Builtins` class doesn't create the `ModuleScopeBuilder` itself
- it _gets the builder_ and can fill it
- it also doesn't close the builder by calling `.build()` - the `Module` does that its 
- this way we can move control over lifecycle of _scope builder_ into a single place - e.g. `Module` itself